src/components/TrackCard.tsx

```
@@ -12,57 +13,74 @@ function isValidArtworkUrl(url: string | undefined): url is string {
+            fill
+            className="object-cover select-none"
+            style={{ WebkitUserDrag: "none" }}
+            draggable={false}
+            onDragStart={(e) => e.preventDefault()}
+            unoptimized
           />
-        </a>
-      ) : null}
+        ) : (
+          <div className="flex h-full w-full items-center justify-center" draggable={false} onDragStart={(e) => e.preventDefault()}>
+            <span className="text-4xl opacity-20">🎵</span>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)